data/web-workers.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @title Web Workers
+ * @difficulty intermediate
+ * @tags cli, web
+ * @run <url>
+ * @resource {https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers} MDN: Web Workers
+ * @resource {https://deno.land/manual/runtime/workers} Deno: Workers
+ *
+ * Workers are the only ways of running javascript off of the main thread.
+ * This can be useful for a wide variety of programs, especiall those where
+ * there is a lot of computation that needs to be done without blocking a
+ * thread.
+ */
+
+// File: ./main.ts
+
+// Currently, Deno only suppors module-type workers. To instantiate one
```

Comment:
```suggestion
// Currently, Deno only supports module-type workers. To instantiate one
```

Reply:
I swear I was not drunk writing this. I'm blaming my keyboard on this one.